deployment/k8s/charts/aggregates/migrationAssistantWithArgo/templates/resources/s3/createS3SnapshotBucket.yaml

```diff
@@ -0,0 +1,92 @@
+# templates/create-s3-bucket-job.yaml
+{{- if .Values.snapshotBucketConfiguration.create }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: create-s3-bucket-{{ .Release.Name }}
```

Comment:
Hoping ACK can handle this logic for us both in EKS and locally with LocalStack but we will see